web/app_options.js

```diff
@@ -526,4 +538,26 @@ class AppOptions {
   }
 }
 
+if (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC")) {
+  AppOptions._checkDisablePreferences = () => {
+    if (AppOptions.get("disablePreferences")) {
+      // Give custom implementations of the default viewer a simpler way to
+      // opt-out of having the `Preferences` override existing `AppOptions`.
+      return true;
+    }
+    for (const name in userOptions) {
+      // Ignore any compatibility-values in the user-options.
+      if (compatibilityParams[name] !== undefined) {
+        continue;
+      }
+      console.warn(
+        "setAll: The Preferences may override manually set AppOptions; " +
```

Comment:
Should we remove `setAll: ` here given that this code is no longer in that method?

Reply:
Right, good catch.